src/App.tsx

```diff
@@ -15,47 +15,130 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+type SortCallback = (a: string, b: string) => number;
+
+enum SortType {
+  LENGTH = 'length',
+  ALPHABETICAL = 'alphabetical',
+  NONE = '',
+}
+
 export const App: React.FC = () => {
+  const [goods, setGoods] = useState<string[]>(goodsFromServer);
```

Comment:
In this task, you are not supposed to keep goods in state

Reply:
Take the same approach as in the previous task - list-of-goods (that one without hooks)